main.go

```diff
@@ -951,9 +951,9 @@ func _main(ctx context.Context) error {
 	// Enable new blocks being stored into the base DB's cache.
 	baseDB.EnableCache()
 
-	// Deactivate displaying the sync status page after the db sync was completed.
+	// Block further usage of the barLoad by sending a nil value
 	if barLoad != nil {
-		close(barLoad)
+		barLoad <- nil
```

Comment:
This will hang if: (1) sync hit an error and bailed out, or (2) `barLoad` is a nil channel (when `!displaySyncStatusPage`).  Please do one of these so it won't block:

```go
select {
	case barLoad <- nil:
	default:
}
```